.github/workflows/lint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           python-version: 3.7
       - name: Install project dependencies
         run: |
-          pip install tensorflow==2.1.0
+          pip install tensorflow-cpu==2.2.0
```

Comment:
Are we always linting on `tensorflow==2.latest-1.0`?

Reply:
I changed it to 2.2 in order to remove some pytype errors that where caused by the way 2.1 handled some imports.